src/database.rs

```diff
@@ -15,6 +15,6 @@ pub async fn create_conn_pool() -> Result<PgPool, Error> {
         Ok(pool) => pool,
         Err(_) => PgPool::connect(FALLBACK_DB_URL).await?,
     };
-    sqlx::migrate!().run(&pool).await?;
+    //sqlx::migrate!().run(&pool).await?;
```

Comment:
Why is this commented?

Reply:
I had some problems running the server several times ( the bug with this line uncommented, but didn't dive into it before

Reply:
The problem was probably that you ran the server for the first time when the new sql was in `1_users.sql` instead of a new migration, so changing it to `2_projects.sql` had conflicts for you. On the main server however it would be the opposite. I would keep this line

Reply:
I believe that was quite much exactly what happend. Anyway, migration line is uncommented now.